packages/playground-examples/copy/ja/3-7/Fixits/Big number literals.ts

```diff
@@ -0,0 +1,34 @@
+//// { compiler: {  target: 99 }, order: 1 }
+
+// JavaScript を書く際に、JavaScript で表現できる数字の大きさに
+// 制限があることをあることをご存知でしたか？
+
+const maxHighValue = 9007199254740991
+const maxLowValue = -9007199254740991
+
+// これらの数字を上回る/下回ることは、大変危険です。
+
+const oneOverMax = 9007199254740992
+const oneBelowMin = -9007199254740992
+
+// このサイズの数値を扱うための解決策は、
+// これらの数値を、number の代わりに BigInts に変換することです:
+//
+// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt
+
+// TypeScriptは、2の52乗以上の数値リテラル(正/負)の修正機能を提供するようになり、
```

Comment:
aboveはmore thanと同じなので以上ではないんですよね。。。
- ref: https://www.b-cafe.net/newsletter/2014/07/%E9%A0%AD%E3%81%AE%E6%95%B4%E7%90%86%E3%81%AB%E3%81%AA%E3%82%8A%E3%81%BE%E3%81%99%EF%BC%81%E3%80%8C%EF%BD%9E%E4%BB%A5%E4%B8%8A%E3%80%8D%E3%82%92%E3%81%82%E3%82%89%E3%82%8F%E3%81%99over-above-more.php
- ref: https://www.ldoceonline.com/jp/dictionary/above
```suggestion
// TypeScriptは、2の52乗より大きいの数値リテラル(正/負)の修正機能を提供するようになり、
```